src/app/api/core/services/policies.service.ts

```diff
@@ -10,11 +10,11 @@ import httpStatus from 'http-status'
 export class PoliciesService extends BaseService {
   private defaultPolicies: Record<UserRole.Client, Record<Resource, UserAction[]>> = {
     [UserRole.Client]: {
-      [Resource.Tasks]: [UserAction.Read],
+      [Resource.Tasks]: [UserAction.Read, UserAction.Update],
```

Comment:
We shouldn't allow client user to update tasks directly since would also allow them to edit title, description, etc.

For this particular issue it would be better not add `UserAction.Update` to the permissions, rather find a workaround for it in the service layer